whylog_vim/input_reader/__init__.py

```diff
@@ -0,0 +1,19 @@
+import re
+from whylog_vim.input_reader.consts import Input, RegexPatterns
+
+
+def filter_comments(content):
+    pattern = re.compile(RegexPatterns.COMMENTS)
```

Comment:
Const for regex string is a good idea, but you still compile regex every time when you invoke this method. Should be module const ```python 
COMMENTS = re.compile(RegexPatterns.COMMENTS) 
```